### ISSUE TITLE
Qubit Rotation in JAX

### ISSUE THREAD

**Issue by araza6** [qgrad/qgrad#1]
Since the idea of qgrad is to extend autodiff capabilities using the existing autodiff libraries (like [JAX](https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html)) to a more general set of functions (quantum hamiltonians as `Qobj` in this case), a good first step is to produce a working tutorial of something in JAX that uses autograd. [Pennylane](https://pennylane.ai/) qubit rotation [tutorial](https://pennylane.ai/qml/demos/tutorial_qubit_rotation.html) seems like a good first step. Pennylane also uses autograd wrapper btw. 

I am working on a notebook to cover this.

**Comment by nathanshammah** [comment]
This may also be related, a notebook in Pennylane on VQE with [quantum natural gradient](https://pennylane.ai/qml/demos/tutorial_vqe_qng.html), also showing it on the QuTiP Bloch sphere cc @quantshah.

**Comment by araza6** [comment]
Closing this on PR https://github.com/qgrad/qgrad/pull/10
